ivy/functional/frontends/jax/devicearray.py

```diff
@@ -137,6 +137,10 @@ def sort(self, axis=-1, order=None):
             axis=axis,
             order=order,
         )
+    
+    def imag(self, /):
+        return jax_frontend.numpy.imag(self._ivy_array)
```

Comment:
```suggestion
        return jax_frontend.numpy.imag(self)
```

I think this should fix your test :)